.github/workflows/percy.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Percy'
+on:
+  push:
+    paths:
+      - 'packages/UI/.storybook/**'
+      - 'packages/UI/src/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Node 16.x  
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
+      - name: Install @percy/cli and @percy/storybook
+        run: yarn add @percy/cli @percy/storybook
+        
+      - name: Percy Storybook Snapshots
+        run: yarn percy storybook https://632ca21aeea1618b7fa0bf48-axgzbckakp.chromatic.com/

```

Comment:
line 24 should not be fixed and should have a dynamic value from the chromatic CI job

Reply:
having a fixed value of https://632ca21aeea1618b7fa0bf48-axgzbckakp.chromatic.com/ means percy will only take snapshot of that storybook always

Reply:
Updated